### ISSUE TITLE
UnhandledPromiseRejectionWarning: Error: incorrect header check

### ISSUE THREAD

**Issue by dnm13** [H2Owater425/node-hitomi#26]
Had a version jump, so I don't actually know from which version this error appears (from v3.0.1 to v3.1.0)
```javascript
(node:30432) UnhandledPromiseRejectionWarning: Error: incorrect header check
    at Zlib.zlibOnError [as onerror] (zlib.js:181:17)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:30432) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which 
was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:30432) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

code:
```typescript
import hitomi from 'node-hitomi'
hitomi.getGalleryData(2008107)
  .then(async (gallery) => {
    console.log(gallery);
})
```
just a guess, but probably ever from dependency change from node-fetch to https

**Comment by H2Owater425** [comment]
When I removed the package's dependency from node-fetch, requesting function that I made was getting a compressed response (as used to). So I had to use an uncompressing module to get raw data from Hitomi (and it was Zlib).
But I soon realized with some benchmarks I made that getting a raw response was way faster than getting a compressed one.
It could be a little annoying, but I want you to use the latest version of the package.
